geo_waypoints:
  - { lat: 37.225350, lon: -121.873700, alt: 15.0 }
  - { lat: 37.225700, lon: -121.873400, alt: 15.0 }
  - { lat: 37.226000, lon: -121.873800, alt: 15.0 }
